src/gameScreens/analysis/useAnalysis.ts

```diff
@@ -0,0 +1,50 @@
+import { useEffect } from "react";
+import {
+  useCurrentAnalysis,
+  useDealtHand,
+  useSetAnalysisState,
+  useStage,
+  useVariant,
+} from "../../redux/hooks";
+import { Stages } from "../../redux/types";
+import { evaluateHand } from "../../strategy/evaluateHand";
+import { VARIANT } from "../../types/variant";
+
+export const useContinuousAnalysis = () => {
```

Comment:
...calculate analysis table in its own dedicated hook, that's saved to the redux store, so that it's available in any component immediately without re-calculation